src/backend/marsha/core/defaults.py

```diff
@@ -58,3 +58,34 @@
 # FLAGS
 SENTRY = "sentry"
 BBB = "BBB"
+
+# Emails sent for scheduled webinars
+REMINDER_1, REMINDER_2, REMINDER_3, REMINDER_IS_STARTED, REMINDER_ERROR = (
+    "REMINDER_1",
+    "REMINDER_2",
+    "REMINDER_3",
+    "REMINDER_IS_STARTED",
+    "REMINDER_ERROR",
+)
+# keys for REMINDERS_STEP
+REMINDER_KEY_STARTS_IN_S, REMINDER_KEY_REGISTER_BEFORE_S, REGISTER_EXCLUDE_STEP = (
+    "STARTS_IN_S",
+    "REGISTER_BEFORE_S",
+    "REGISTER_EXCLUDE_STEP",
+)
+REMINDERS_STEP = {
+    REMINDER_1: {
+        REMINDER_KEY_STARTS_IN_S: 5 * 60,  # 5 minutes before
+        REMINDER_KEY_REGISTER_BEFORE_S: 3 * 60 * 60,  # three hours before
```

Comment:
```suggestion
        REMINDER_KEY_REGISTER_BEFORE_S: 3 * 60 * 60,  # three hours before
        REGISTER_EXCLUDE_STEP: [],
```

Reply:
Doing this you can remove the if statement in the command

Reply:
I still will have an if to check its length not to add a query that filters on nothing but true I can add it here not to make exceptions on the configuration and have all keys declared.